.github/workflows/test-build.yml

```diff
@@ -25,8 +25,6 @@ jobs:
        - name: Run tests
          run: |
           go test ./... -count=1 -race -cover
-         if: matrix.go_version != 1.13
-         continue-on-error: true
 
        - name: Build
          if: matrix.go_version == 1.13
```

Comment:
Could you remove also this line? It's a part of an experiment with github actions. Now i'm planning to try the circle ci pipeline instead

Reply:
But the tests weren't executed; I simply tried removing it.